extension/persistence/eclipselink/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreSessionImpl.java

```diff
@@ -762,4 +769,11 @@ public void rollback() {
       session.getTransaction().rollback();
     }
   }
+
+  // TODO support pagination
```

Comment:
I'd leave it out - EclipseLink is deprecated, the dev effort would be lost.

Reply:
Good point, let me update this comment